### ISSUE TITLE
`gopass sync` works without ssh key in ssh agent

### ISSUE THREAD

**Issue by revengel** [gopasspw/gopass#2257]
### Summary

`gopass sync` command works without ssh key in ssh agent

Please help me to understand the issue

### Steps To Reproduce

* clear `ssh-agent` - `ssh-add -D`
* adding ssh key in `ssh-agent` for gopass syncing `ssh-add /path/to/private/ssh-key`
* try to generate some password `gopass generate some/password/path1/password`
* make `gopass sync`
* password has been generated and pushed to git-server
* clear `ssh-agent` - `ssh-add -D`
* There are no any ssh-key in ssh-agent now. Synchronization should not work now.
* try to generate another password `gopass generate some/password/path2/password`
* make `gopass sync`
* But now the sync has happened again and the new password has been pushed to git server
* At the same time `git clone`, `git pull`, `git push` fall with an error `Please make sure you have the correct access rights`
* ???

### Version

`gopass 1.14.3-git+HEAD go1.18.2 linux amd64`

**Comment by dominikschulz** [comment]
Yes, that does make sense. We use an persistent ssh control Socket to speed up subsequent sync operations: https://github.com/gopasspw/gopass/blob/master/internal/backend/storage/gitfs/ssh_others.go

You can disable that by overriding GIT_SSH_COMMAND.
